enclave-runtime/src/ocall/attestation_ocall.rs

```diff
@@ -115,6 +111,60 @@ impl EnclaveAttestationOCallApi for OcallApi {
 		Ok((qe_report, quote_vec))
 	}
 
+	fn get_dcap_quote(&self, report: sgx_report_t, quote_size: u32) -> SgxResult<Vec<u8>> {
+		let mut return_quote_buf = vec![0u8; quote_size as usize];
+		let p_quote = return_quote_buf.as_mut_ptr();
+		let p_report = &report as *const sgx_report_t;
```

Comment:
Wow these pointer transforms are a bit nasty. I have looked a bit around, but it seems there is now ergonomic way of doing this. The only alternative I found was:

```rust
let p_report: *const sgx_report_t = &report;
```

Which is not better IMO.